### ISSUE TITLE
Node v15 - Help wanted

### ISSUE THREAD

**Issue by SimonSchick** [DefinitelyTyped/DefinitelyTyped#49497]
Is something wrong with CI or publishing infrastructure?

- Check the [Infrastructure status updates](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/44317) thread for known issues.
- Leave a message on the [DefinitelyTyped Gitter](http://gitter.im/DefinitelyTyped/DefinitelyTyped).

If you know how to fix the issue, make a pull request instead.

- [x] I tried using the `@types/node` package and had problems.
- [ ] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [ ] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @Flarna @ExE-Boss (so many more but the list is stupidly long)

I'm looking for some people to coordinate a structured update to node v15(.2) with, the changelog is quite exhaustive and I currently don't have a whole lot of time to do [those](https://github.com/DefinitelyTyped/DefinitelyTyped/pulls?q=is%3Apr+author%3ASimonSchick+node) on my own (due to a new job).

tl;dr
- Looking for 2-3 maintainers
- Looking to equally split work load by modules to avoid conflicts.

I believe this effort is important to prevent the node typings from falling behind node itself.

**Comment by ExE-Boss** [comment]
See PR <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/48981>

**Comment by SimonSchick** [comment]
This was done.
